docs/howto/openstack/nova/config-drive.md

```diff
@@ -20,35 +20,64 @@ configuration source.
 ## Store metadata on a configuration drive
 
 A **configuration drive** (config drive) is a read-only virtual drive
-that is attached to a server during boot. The server can then
+that gets attached to a server during boot. The server can then
 mount the drive and read files from it. Configuration drives are used
 as a data source for
 [cloud-init](https://cloudinit.readthedocs.io/en/latest/).
 
 
-## Enable the configuration drive on server creation (`openstack` CLI)
+## Enable the configuration drive on server creation
 
-To enable the configuration drive, you need to pass the parameter
-`--use-config-drive` to the `openstack server create` command.
+Follow our [How-To guide](new-server.md) to create your new cloud server
+using either the {{gui}} or the OpenStack CLI.
 
-In the following example, replace the image, flavor, keypair, and
-network reference, as well as the server name, to match your desired
-configuration.
+=== "{{gui}}"
+    When you are done configuring the server, right before creating it
+    notice the icon on the left-hand side of _Advanced Options_. Click on it
+    to expand all related options.
 
-```bash
-openstack server create \
-  --use-config-drive \
-  --image "Ubuntu 20.04 Focal Fossa" \
-  --flavor b.1c2gb \
-  --keypair mykey
-  --nic net-id=3a747038-ee59-404c-973d-5f795e8ebb73 \
-  myserver
-```
+    ![Expand advanced configuration options](assets/config-drive/shot-01.png)
 
-Once the server launches, you can monitor its configuration process
-by monitoring the server console log:
+    In the _User-Data_ area, you see everything that will be executed upon
+    server boot.
 
-```bash
-openstack console log show myserver
-```
+    Right below, in the _User-data propagation method_ area, the _Use
+    metadata service (Default)_ option is preselected for you. To ignore
+    that and go for the alternative metadata discovery method, select _Use
+    configuration drive_.
 
+    ![Select the configuration drive method](assets/config-drive/shot-02.png)
+
+    Then, instantiate the server by clicking the green _Create_ button.
+
+    Once the server is ready, you may see what happened during configuration
+    by examining the console log. In the {{gui}}, click on the server line
+    to get an extended view of its characteristics, and then click once more
+    on the _Console Output_ tab.
+
+    ![Watch the server console](assets/config-drive/shot-03.png)
```

Comment:
Ending the page with that screenshot feels weird. It looks as there is an explanatory sentence missing after this.

Reply:
I can add a closing sentence there.